src/lib/schematics/address-form/index.ts

```diff
@@ -30,7 +30,7 @@ export default function(options: Schema): Rule {
  */
 function addFormModulesToModule(options: Schema) {
   return (host: Tree) => {
-    const modulePath = findModuleFromOptions(host, options);
+    const modulePath = findModuleFromOptions(host, options)!;
```

Comment:
This is never `undefined` or `null` because the `addXModulesToModule` methods are only called if `skipImport` is set to `false`. And the `findModuleFromOptions` method only returns undefined if that option is enabled.

See: https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/utility/find-module.ts#L26

@jelbourn Want me to add a comment for this (afraid of repeating it for every schematic though)

Reply:
Could the function return an empty string in that case?

Reply:
What would be the benefit? This comes from an utility function from `@schematics/angular`. It should be really *never* undefined.